community/portworx/templates/grafana-dashboard.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and .Values.csiCloudDrive (eq .Values.csiCloudDrive true)}}
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: grafana-dashboards
+  namespace: kube-system
```

Comment:
Same comment regarding namespace of kube-system.

Reply:
reply same as above.